packages/web-react/src/components/Toast/Toast.tsx

```diff
@@ -4,8 +4,15 @@ import { useStyleProps } from '../../hooks';
 import { SpiritToastProps } from '../../types';
 import { useToastStyleProps } from './useToastStyleProps';
 
+const defaultProps: SpiritToastProps = {
+  alignmentX: 'center',
```

Comment:
Can we use a constant from the dictionary instead of `center`, please? We are repeating the defaults.

Reply:
done